contract_manager/package.json

```diff
@@ -23,20 +23,32 @@
   "dependencies": {
     "@certusone/wormhole-sdk": "^0.9.8",
     "@coral-xyz/anchor": "^0.29.0",
+    "@cosmjs/cosmwasm-stargate": "^0.32.3",
+    "@cosmjs/stargate": "^0.32.3",
     "@injectivelabs/networks": "^1.14.6",
     "@mysten/sui.js": "^0.49.1",
+    "@pythnetwork/client": "^2.21.0",
+    "@pythnetwork/contract-manager": "workspace:*",
     "@pythnetwork/cosmwasm-deploy-tools": "*",
     "@pythnetwork/entropy-sdk-solidity": "*",
     "@pythnetwork/price-service-client": "*",
```

Comment:
why are these not changed?
and also why for some we've `workspace:^` and for some `workspace:*`?

Reply:
I actually didn't manually add these, they came in by running `pnpm add`.  However, they are documented at https://pnpm.io/workspaces#publishing-workspace-packages -- basically, they affect how the dependency versions get modified when publishing.  Upon publishing, uses of `workspace:*` will be replaced with an exact version with no prefix (i.e. `1.0.0`), where `workspace:^` will be replaced with a carat-prefixed dependency (i.e. `^1.0.0`).

If we're using semver correctly then presumably `workspace:^` is the more semantically correct way to do this, and probably should be the default.  I'm not sure what the heuristic is that the pnpm cli uses to select which to use when running `pnpm add` but IMO we should leave it as-is for this release and should audit these later.